scripts/should-rebuild-on-vercel.sh

```diff
@@ -95,9 +93,9 @@ get_all_changed_files () {
         const data = require("./diff-since-main.json");
 
         // If some commits on this branch have the phrase "[no vercel]" or
-        // "[skip examples]" or some combination like that in their message, skip
+        // "[skip deploy]" or some combination like that in their message, skip
```

Comment:
LOL, I forgot we had this! Need to remember, because this is useful to save carbon emissions! 🌍

Reply:
Same 😅